pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin_test.go

```diff
@@ -459,17 +451,24 @@ var _ = Describe("MeshTimeout", func() {
 						},
 					}},
 				},
+			}).WithPolicies(xds.MatchedPolicies{
+			Dynamic: map[core_model.ResourceType]xds.TypedMatchingPolicies{
+				api.MeshTimeoutType: {
+					Type:    api.MeshTimeoutType,
+					ToRules: given.toRules,
+				},
 			},
-			RuntimeExtensions: map[string]interface{}{},
-		}
-		Expect(gateway_plugin.NewPlugin().(core_plugins.ProxyPlugin).Apply(context.Background(), xdsCtx.Mesh, &proxy)).To(Succeed())
+		}).
+			Build()
```

Comment:
Nit: I would put `WithPolicies(...` to new line

Reply:
done